skglm/solvers/prox_newton.py

```diff
@@ -15,7 +19,7 @@ class ProxNewton(BaseSolver):
     p0 : int, default 10
         Minimum number of features to be included in the working set.
 
-    max_iter : int, default 20
+    max_iter : int, default 50
```

Comment:
Choosing default values is hard but I don't think 20 is a bad value, it prevents the solver from running too long and people are usually interested in a moderately accurate solution.
Are there reasons to change this ?

Reply:
I don't have a strong opinion on this change.
Adding a warning is indeed sufficient.